Sources/Core/ObjectiveCDebugExtension.swift

```diff
@@ -10,22 +10,42 @@ import Foundation
 
 extension ObjCModelRenderer {
     func renderDebugDescription() -> ObjCIR.Method {
-        let props = properties.map { (param, prop) -> String in
+        let props = properties.filter { (_, schema) -> Bool in
```

Comment:
This code currently walks `properties` twice: first to act on the non-boolean properties and then to act on the boolean properties. It would be nice to restructure this to do that filtering in one pass. I'm not sure how easy that is using Swift's functional primitives, though.

Reply:
I'll have to try this later.